pupil_src/shared_modules/pupil_detectors/__init__.py

```diff
@@ -9,18 +9,17 @@
 ---------------------------------------------------------------------------~(*)
 """
 
-# when running from source compile cpp extension if necessary.
-import sys
+try:
+    from .detector_2d import Detector_2D
+except ModuleNotFoundError:
+    # when running from source compile cpp extension if necessary.
+    import sys
+    if not getattr(sys, "frozen", False):
```

Comment:
This check can be removed since the build detector should always be present in the bundle and therefore not fail. Same applies to the `optimization_calibration/` changes.

Reply:
Updated and rebased.